travis-ci/vmtest/run_selftests.sh

```diff
@@ -43,9 +43,9 @@ fi
 
 cd libbpf/selftests/bpf
 
-test_progs
+[[ "$(uname -m)" != s390x ]] && test_progs
```

Comment:
Hm, this doesn't look right: should be `[[ "$(uname -m)" == s390x ]] || test_progs`